presto-iceberg/src/main/java/com/facebook/presto/iceberg/IcebergAbstractMetadata.java

```diff
@@ -612,6 +641,12 @@ protected ImmutableMap<String, Object> createMetadataProperties(Table icebergTab
         properties.put(METADATA_DELETE_AFTER_COMMIT, IcebergUtil.isMetadataDeleteAfterCommit(icebergTable));
         properties.put(METRICS_MAX_INFERRED_COLUMN, IcebergUtil.getMetricsMaxInferredColumn(icebergTable));
 
+        SortOrder sortOrder = icebergTable.sortOrder();
+        // TODO: Support sort column transforms (https://github.com/trinodb/trino/issues/15088)
```

Comment:
We should probably add this caveat to the documentation as well as file our own issue in the presto issue tracker and link that instead of Trino's

Reply:
Created an  issue in presto issue tracker.
 https://github.com/prestodb/presto/issues/24250